RobotsWantedLeague/RobotsWantedLeague.csproj

```diff
@@ -6,4 +6,13 @@
     <ImplicitUsings>enable</ImplicitUsings>
   </PropertyGroup>
 
+  <ItemGroup>
```

Comment:
This adds mongodb as a dependency to the project.  This file was changed automagically via 

```shell
dotnet add package MongoDB.Driver
```